container_pipeline/utils.py

```diff
@@ -211,3 +211,29 @@ def complete(self):
             os.remove(self._path)
             self.logger.info('Removing lock file for {} at {}'.format(
                 self.name, self._path))
+
+
+def form_targetfile_link(git_URL, git_path, git_branch, target_file):
+    """
+    Helper function to generate Dockerfile link.
+    :param git_URL: The url of the git repository.
+    :param git_path: The path, relative to the git repository root,
+                     where the file resides.
+    :param git_branch: The repository branch where the file resides.
+    :param target_file: The name of the target file.
+    :return: The reachable link to the dockerfile.
+    """
+
+    # TODO : Move this into a lib.
+
+    link_url = None
+    if "github" in git_URL or "gitlab" in git_URL:
```

Comment:
What happens when someone's not using either GitHub or GitLab? What if they're using BitBucket instead? We return `None` as the value of `link_url`, right? Is that valid for the pieces that make use of this function?

Reply:
That is something i need to think about, going forward,.

Reply:
You mean not to be worked upon in this PR?

Reply:
yes, but i will update it as soon as i can